examples/consumer/main.go

```diff
@@ -383,8 +429,23 @@ func pullEvents() {
 			return
 		}
 	}
+	allFailed := true
 	for _, s := range subs {
-		getCurrentState(s.Resource)
+		if err := getCurrentState(s.Resource); err == nil {
+			allFailed = false
+		} else {
+			log.Errorf("%s", err)
```

Comment:
Couldn't this be `log.Error(err)` or if it needs to be a string `log.Error(err.Error())`